src/components/FileMenu/GetLinkDialog.js

```diff
@@ -17,7 +17,9 @@ export const GetLinkDialog = ({ type, id, onClose }) => {
     const { baseUrl } = useConfig()
 
     // TODO simply use href from the visualization object?
-    const appUrl = new URL(appPathFor(type, id), baseUrl)
+    const appBaseUrl = new URL(baseUrl, self.location.href)
```

Comment:
What about combining these into a single function with args (baseUrl, self.location.href) and put it in utils, and add a few jest tests for it?

Reply:
The TODO line above is still there because I think all of this can be simplified by just returning the `href` prop in the AO.
I think that should work with different instances, basically the origin should reflect the instance in use.
For that tho we need to make sure all apps fetch the `href` prop.